### PR TITLE
[ioctl] Dev account ethaddr

### LINKED ISSUE #3033: [ioctl] build `account ethaddr` command line into new ioctl
The new ioctl (./ioctl/newcmd) is a refactored version of the old ioctl(./ioctl/cmd). It includes two major changes:
- The [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) is passed when constructing the Cobra command.
- Unit tests are to be added for each command.

The newcmd code currently is a draft. The code of old ioctl can be used for reference.

### What would you like to be added
add `accountethaddr.go` and `accountethaddr_test.go` in `./ioctl/newcmd/account`. 
**Code for reference**:
https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/account/accountethaddr.go

### PR DESCRIPTION
**Pull request by LuckyPigeon**
fix #3033

### CODE REVIEW

**Comment by huof6829** [comment]
add sub command in NewAccountCmd of account.go

File: ioctl/newcmd/account/accountethaddr.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			client.PrintInfo(message.String())
+			return nil
+		},
+	}
+}
+
+type ethaddrMessage struct {
+	IOAddr  string `json:"ioAddr"`
+	EthAddr string `json:"ethAddr"`
+}
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountethaddr.go:74]: we may delete this struct.

File: ioctl/newcmd/account/accountethaddr_test.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+)
+
+func TestNewAccountEthAddr(t *testing.T) {
+	require := require.New(t)
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	client := mock_ioctlclient.NewMockClient(ctrl)
+	client.EXPECT().SelectTranslation(gomock.Any()).Return("", config.English).AnyTimes()
+	client.EXPECT().PrintInfo(gomock.Any()).Do(func(info string) {
+		fmt.Println(info)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountethaddr_test.go:30]: delete

**Review by CoderZhi — APPROVED** [review]

File: ioctl/newcmd/account/accountethaddr.go (hunk 1 above)
Comment (LuckyPigeon) [reply ioctl/newcmd/account/accountethaddr.go:74]: But what should we print at the end of command

File: ioctl/newcmd/account/accountethaddr_test.go (hunk 2 above)
Comment (LuckyPigeon) [reply ioctl/newcmd/account/accountethaddr_test.go:30]: Just a simulation for what `client.PrintInfo` do in `accountethaddr.go` L65.
We can do this after delete `ethaddrMessage` struct and figure out what should print at the end of command.

File: ioctl/newcmd/account/accountethaddr.go (hunk 1 above)
Comment (LuckyPigeon) [reply ioctl/newcmd/account/accountethaddr.go:74]: how about something like `output.PrintResult(ethaddr)`

**Pull request merged**